File: 1431. Kids With the Greatest Number of Candies/solution1.py
class Solution:
    def kidsWithCandies(self, candies: List[int], extraCandies: int) -> List[bool]:
        max = 0
        for x in candies:
            if x > max:
                max = x
        lst=[]
        for x in candies:
            if (x+extraCandies) >= max:
                lst.append(True)
            else:
                lst.append(False)
        
        return lst

File: 1431. Kids With the Greatest Number of Candies/solution1.java
class Solution {
    public List<Boolean> kidsWithCandies(int[] candies, int extraCandies) {
        List<Boolean> arr = new ArrayList<>();
        var max = 0;
        for (int i:candies){
            if(i > max) max = i;
        }
        
        for (int i=0 ; i <candies.length;i++){
            if(candies[i]+extraCandies >= max) 
            {
                arr.add(true);
            }else
            {
                arr.add(false);
            }
        }
        return arr;
    }
}
